README.md

```diff
@@ -488,6 +491,21 @@ The following is the list of connection options and default values.
 | `yieldTime`            |                           | If set and processing exceeds yieldTime, client will yield to IO callbacks before processing additional inbound messages 
 
 
+### Jitter 
+
+The settings `reconnectTimeWait`, `reconnectJitter`, `reconnectJitterTLS`, `reconnectDelayHandler` are all related.
+They control how long before the NATS client attempts to reconnect to a server it has previously connected.
+
+The intention of the settings is to spread out the number of clients attempting to reconnect to a server over a period of time, 
+and thus preventing a ["Thundering Heard"](https://docs.nats.io/developing-with-nats/reconnect/random).
```

Comment:
It's "herd".. missed that in the other PR..

Reply:
Fixed here. Will fix the other.